mmeds/server.py

```diff
@@ -438,7 +438,13 @@ def run_validate(self, myMetaData, simplified):
         cp.log('before validator run')
 
         # Check the metadata file for errors
-        errors, warnings, subjects = valid.run()
+        try:
+            errors, warnings, subjects = valid.run()
+        except Exception as ex:
```

Comment:
oh yeah, forgot about this. Do want the try catch to log the exception though, otherwise the validator doesn't actually report what column is causing the validation error

Reply:
the exception does get handled in the next block, but the try except makes sure it gets logged. Adding a todo because this is sub-optimal